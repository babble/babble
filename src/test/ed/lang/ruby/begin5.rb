
def foo z
  puts "foo"
  puts z
  raise 5
end

def save
  puts "save"
end

def bar
  return ( foo 6 rescue save )
end

bar
