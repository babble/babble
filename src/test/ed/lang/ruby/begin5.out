foo
6
save
111
foo
7
6
